src/note-c/.github/workflows/ci.yml

```diff
@@ -124,6 +124,23 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           path-to-lcov: ./build/test/coverage/lcov.info
 
+  run_low_mem_unit_tests:
```

Comment:
Hmmm, this shouldn't be here. I'll change the update_note_c.sh script to delete src/note-c/.github in addition to the other folders we're already culling.